src/themes/motoScout24.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  createSystem,
+  defaultBaseConfig,
+  mergeConfigs,
+} from '@chakra-ui/react';
+
+import { sharedConfig } from './shared';
+
+const motoScout24Config = mergeConfigs(
+  {
+    theme: {
+      semanticTokens: {
+        colors: {
+          brand: {
+            primary: { value: '{colors.brand.400}' },
+            btnShadow: { value: '{colors.brand.600}' },
+          },
+        },
+      },
+      tokens: {
+        colors: {
+          brand: {
+            50: { value: '#FFF8F9' },
+            100: { value: '#FFE4E5' },
+            200: { value: '#FFC0C2' },
+            300: { value: '#FD7978' },
+            400: { value: '#FF4C52' },
+            500: { value: '#EF474D' },
+            600: { value: '#D64045' },
+            700: { value: '#A83236' },
+            800: { value: '#7D2528' },
+            900: { value: '#5F1C1E' },
+          },
+        },
+      },
+    },
+  },
+  sharedConfig,
```

Comment:
This is a workaround.
I would much rather prefer to merge brand-specific config into shared. However, it results in values being added/overwritten in the `sharedConfig`. So if both themes define `brand` colour, the last one would be applied to both themes (i.e. both AS and MS themes would have MS brand colour)

Reply:
Fyi, I opened an [issue](https://github.com/chakra-ui/chakra-ui/issues/9193) on `chakra-ui`